pkg/systemd/generate/containers.go

```diff
@@ -475,7 +475,7 @@ func executeContainerTemplate(info *containerInfo, options entities.GenerateSyst
 				return "", err
 			}
 		}
-		info.ExecStart = strings.Join(startCommand, " ")
+		info.ExecStart = strings.Join(startCommand, " \\\n\t")
```

Comment:
I don't like this syntax, I would prefer if only options are separated with newlines and not the values, i.e.
```
--name myname \
-d \
-p 8080
```

You current code would look like:
```
--name \
myname \
-d \
-p \
8080 \
```

Reply:
Any clue how can I achieve this?

Reply:
Only add new line before fields that start with a `-`